R/utils.R

```diff
@@ -131,3 +131,62 @@ myhashmap_getvec_keyint <- function(mapped, keys) {
   keys <- as.character(keys) # key should be a string
   return(unlist(lapply(keys, function(x) {mapped$get(x)}), use.names = FALSE, recursive = FALSE))
 }
+
+#' Convert a quanteda dictionary to keywords
+#'
+#' This function converts or reads a dictionary object from quanteda to a named list. "Glob"-style wildcard expressions (e.g. politic*) are resolved based on the available terms in your texts.
+#' @param file file identifier for a foreign dictionary, e.g. path to a dictionary in YAML or LIWC format
+#' @param docs texts read via [keyATM_read()]
+#' @param dictionary a quanteda dictionary object, ignore if file is not NULL
+#' @param split boolean, if multi-word terms be seperated, e.g. "air force" splits into "air" and "force".
+#' @param ... additional parameters for [quanteda::dictionary()]
+#' @return a named list which can be used as keywords for e.g. [keyATM()]
+#' @seealso \code{\link{dictionary}}
+#' @export
+#' @examples
+#' \donttest{
```

Comment:
Can we use `donttrun` instead? Fitting the model for examples takes some time. I would like to keep the example for website's tutorial.

Reply:
It was from my experience dealing with the CRAN people that when an example can run from top to bottom (e.g. the example doesn't depend on says, API key) they would ask you to change it from `dontrun` to `donttest`. It is because, I quote [a response from my previous CRAN submission of another package]: 
> \dontrun{} should only be used if the example really cannot be executed (e.g. because of missing additional software, missing API keys, ...) by the user.

This example can run from top to bottom by the user. But if you can get keyATM through CRAN previously with other runnable examples wrapped in `dontrun`, surely this example can also be wrapped in `dontrun`. CRAN is not always consistent with their policies.

Reply:
Yeah, I remember the PDF manual became ugly if I used `donttest`. In the emails with CRAN maintainers, I told them I test codes with `testthat`.